### ISSUE TITLE
Mysterious "undefined" error when assigning to 'bind'

### ISSUE THREAD

**Issue by kumavis** [Agoric/SES#135]
```js
b = r.evaluate(`class a {};a.bind = 1`)
// => Thrown: undefined
```

**Comment by kumavis** [comment]
object
```js
> b = r.evaluate(`const a = {}; a.toString=1`)
// => Thrown: undefined
```

**Comment by kumavis** [comment]
node v10.16.0
ses@0.5.0

**Comment by kumavis** [comment]
ah, seems like its just a poor error message in node?

nodejs
```
sesify on  master [$!] is 📦 v2.0.0 via ⬢ v10.16.0 
➜ echo 'r = require(`ses`).makeSESRootRealm({ errorStackMode: true }); r.evaluate(`const a = {}; a.toString=1`)' | node
/home/xyz/Development/sesify/node_modules/ses/dist/ses.cjs.js:1812
        throw err2;
        ^

undefined
```

browser via browserify + smokestack
```
sesify on  master [$!] is 📦 v2.0.0 via ⬢ v10.16.0 
➜ echo 'r = require(`ses`).makeSESRootRealm({ errorStackMode: true }); r.evaluate(`const a = {}; a.toString=1`)' | browserify - | smokestack
not ok - Cannot assign to read only property 'toString' of object '#<Object>'
http://localhost:35577/script.js:2105
          throw err2;
           ^

undefined
```

**Comment by erights** [comment]
Yes, this would be the override mistake. Node and Salesforce have already adopted our workaround, of turning these into accessor properties whose behavior emulates a frozen data property as if tc39 had not made the override mistake. 

We should add this workaround back into our own shim. Node applies the workaround to all methods on all primordial prototypes. Salesforce applies it to all methods on five key primordial prototypes, including `Object.prototype`, `Function.prototype`, `Array.prototype`, `Error.prototype`. I forget which the fifth is, but perhaps `RegExp.prototype`?

Attn @jfparadis @jdalton @bmeck @guybedford

**Comment by bmeck** [comment]
I'm neutral on the list of things that get patched, but i agree that having at least an option to enable this would be great.

**Comment by erights** [comment]
I see. 

The fact that it is failing is a SES shim bug in that we need to revive the workaround for the override mistake. 

The fact that it is throwing undefined rather than something sensible is a distinct SES shim bug.

See https://github.com/Agoric/SES/issues/136 for some failures caused by the absence of the workaround.

**Comment by erights** [comment]
IIUC, the Salesforce experience is that after applying the workaround to those five primordial prototypes, the override mistake was masked well enough for practical purposes.

Node is as not sensitive to startup latency as the browser is. If all uses of SES in the browser can afford @guybedford's approach, the SES shim should do that. Otherwise, we should adopt the Salesforce's approach. At least at first, and see what breaks.

**Comment by kumavis** [comment]
> The fact that it is throwing undefined rather than something sensible is a distinct SES shim bug.

just want to clarify, this is the intended focus of this issue
